src/components/SearchBox.vue

```diff
@@ -53,6 +53,7 @@
           type="reset"
           :title="clearTitle"
           :class="suit('reset')"
+          :hidden="!currentRefinement || state.isSearchStalled"
```

Comment:
why does it need to be hidden if the search is stalled?

Reply:
Don't know but it's the behaviour on the other implementation (thoughts @bobylito?). We can merge it as it is (consistency). But we can review the behaviour latter on all the implem, WDYT?

Reply:
yes, that makes sense